docs/test/test-guide.adoc

```diff
@@ -0,0 +1,30 @@
+= Test Guide
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+== Setting up test environment
+
+Requires *Go 1.14*
```

Comment:
Can we clarify _why_ we require Go 1.14?

Is this a minimum, or a hard-requirement?

Reply:
This is minimum otherwise user will hit issue like https://github.com/redhat-developer/kam/issues/22 but not the hard requirement. Uses can use greater version too.

I will add more clarification in the doc

Reply:
Done

Reply:
How about we say "The minimum version of Go required is in the go.mod file" and link to it?

This reduces the cost of updating to newer versions?

Reply:
make sense. Done